SEP-0013.md

```diff
@@ -0,0 +1,198 @@
+        """
+
+    @abc.abstractproperty
+    def data_version(self):
+        """
+        The data version.
+        """
+
+    ##### Observer Coordinate Info #####
+
+    @abc.abstractproperty
+    def observer_location(self):
+        """
+        Coordinate of observatory location.
+        """
+
+    def observer_radial_velocity(self):
+        """
+        Velocity of observer in direction of source.
+        """
+
+    @abc.abstractproperty
+    def celestial_frame(self):
+        """
+        The celestial coordinate frame of the image axes.
+        """
+
+    @abc.abstractproperty
+    def distance_to_sun(self):
+        """
+        Distance to Sun center from observatory.
+        """
+
+    @abc.abstractproperty
+    def light_travel_time(self):
+        """
+        Light travel time from emission source to observer.
+        """
+
+    @abc.abstractproperty
+    def light_travel_time_to_earth(self):
+        """
+        Light travel time from observer to Earth.
+        """
+
+    ##### Time Info #####
+
+    @abc.abstractproperty
+    def date_reference(self):
+        """
+        The base time from which time axis values are measured.
+        """
+
+    @abc.abstractproperty
+    def date_start(self):
+        """
+        Time at which the observation(s) were started.
+        """
+
+    @abc.abstractproperty
+    def date_average(self):
+        """
+        The mean/representative time of the observation(s).
+        """
+
+    @abc.abstractproperty
+    def date_end(self):
+        """
+        The time at which the observation(s) were ended.
+        """
+
+    @abc.abstractproperty
+    def exposure_time(self):
+        """
+        The exposure time(s) of the observation(s).
+        """
+
+    ##### Misc. #####
+
+    @abc.abstractproperty
+    def rsun_meters(self):
+        """
+        Solar radius in units of length.
+        """
+
+    @abc.abstractproperty
+    def rsun_angular(self):
+        """
+        Solar radius in angular units as seen from observatory.
+        """
+
+    ##### Spectral Info #####
+
+    @abc.abstractproperty
+    def spectral_window(self):
+        """
+        The name of the spectral window in which data has been taken.
+        """
+
+    ##### Imaging Info #####
+
+    @abc.abstractproperty
+    def filter(self):
+        """
+        The name of the filter(s) used to make the observation.
+        """
+
+    @abc.abstractproperty
+    def automatic_exposure_control_on(self):
+        """
+        Whether automatic exposure control was on during observations.
+        """
+
+    ##### Operational Info #####
+
+    @abc.abstractproperty
+    def observing_mode(self):
+        """
+        The observing mode used to make the observations.
+        """
+
+    @abc.abstractproperty
+    def observing_mode_id(self):
+        """
+        Identification number of the observing mode.
+        """
+
+    @abc.abstractproperty
+    def SAA(self):
```

Comment:
Redundant given observer position/time?

Reply:
An SAA flag is often provided in FITS headers.  So there is a philosophical question here.  Is this metadata object for the minimal set of metadata from which all others can be derived?  Or is it to represent the metadata attached to the data?  Put another way, do we discard redundant information provided by instrument teams and ask users to recalculate it from other metadata?  If we do, who's to say these will provide the same results?  Where is the boundary of the SAA?  It may depend on the instrument's sensitivity to the SAA.  In this case a general user may come up with a different result to the more knowledgeable instrument team.